### PR TITLE
Enable build for Elasticsearch 2.0

### PR DESCRIPTION
**Pull request by im-denisenko**
Please don't merge yet.

--
[Release notes](https://www.elastic.co/downloads/elasticsearch)

### CODE REVIEW

File: ansible/es-playbook.yml [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
-    - ES_VER: "1.7.1"
-    - ES_SHORT_VER: "1.7"
-    - ES_MAPPER_ATTACHMENTS_VER: "2.7.0"
-    - ES_TRANSPORT_MEMCACHED_VER: "2.7.0"
-    - ES_TRANSPORT_THRIFT_VER: "2.7.0"
-    - ES_GEOCLUSTER_FACET_VER: "0.0.12"
-    - ES_IMAGE_PLUGIN_VER: "1.7.1"
+    - ES_VER: "2.0.0~beta1"
+    - ES_SHORT_VER: "2.0"
+    # - ES_MAPPER_ATTACHMENTS_VER: "3.0.0"
```
Comment (ruflin) [inline ansible/es-playbook.yml]: Are plugins not working anymore so you commented it out?

**Comment by ruflin** [comment]
@im-denisenko Let me know when I should have a closer look.I think there is no travis build because the pull request is not to master.

File: ansible/es-playbook.yml (hunk 1 above)
Comment (im-denisenko) [inline ansible/es-playbook.yml]: There are some breaking changes in plugin api and seems like only mapper attachment is ready for 2.0. I will revert it later.

Sorry I'm quite slow here :)

Comment (ruflin) [inline ansible/es-playbook.yml]: No rush ;-)

**Comment by im-denisenko** [comment]
Build is completely red, but at least is running on every commit now.

@ruflin Could you please review and merge this into `elasticsearch-2.0` so we can move forward step-by-step?

File: lib/Elastica/Exception/ResponseException.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ public function __construct(Request $request, Response $response)
     {
         $this->_request = $request;
         $this->_response = $response;
-        parent::__construct($response->getError());
+        parent::__construct(json_encode($response->getError()));
```
Comment (im-denisenko) [inline lib/Elastica/Exception/ResponseException.php]: This is temporary workaround that fixes fatal error (now getError return array instead of string).

**Comment by ruflin** [comment]
@im-denisenko I merged it into the branch. Good idea to have an open elasticsearch-2.0 branch so everyone can contribute.

**Pull request merged**